drivers/sensor/tdk/icp101xx/icp101xx.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2023 TDK Invensense
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_ICP101XXX_ICP101XXX_H_
+#define ZEPHYR_DRIVERS_SENSOR_ICP101XXX_ICP101XXX_H_
+
+#include <zephyr/types.h>
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/sensor.h>
+#include "Devices\Drivers\ICP101xx\Icp101xx.h"
+#include "Devices\Drivers\ICP101xx\Icp101xxSerif.h"
+
+#define DT_DRV_COMPAT invensense_icp101xx
+
+#define ICP101XX_BUS_I2C DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)
+
+typedef struct {
```

Comment:
We don't typedef structs in zephyr typically

Reply:
Agreed, will be changed.